var/spack/repos/builtin/packages/tensorflow/package.py

```diff
@@ -0,0 +1,291 @@
+            filter_file(r'if workspace_has_any_android_rule\(\)',
+                        r'if True',
+                        'configure.py')
+
+        # additional config options starting with version 1.6
+        if self.spec.satisfies('@1.6.0:'):
+            env['TF_NEED_KAFKA'] = '0'
+
+        # additional config options starting with version 1.8
+        if self.spec.satisfies('@1.8.0:'):
+            env['TF_DOWNLOAD_CLANG'] = '0'
+            env['TF_NEED_AWS'] = '0'
+
+        # additional config options starting with version 1.12
+        if self.spec.satisfies('@1.12.0:'):
+            env['TF_NEED_IGNITE'] = '0'
+            env['TF_NEED_ROCM'] = '0'
+
+        # set tmpdir to a non-NFS filesystem (because bazel uses ~/.cache/bazel)        # noqa: E501
+        # TODO: This should be checked for non-nfsy filesystem, but the current
+        #       best idea for it is to check
+        #           subprocess.call(['stat', '--file-system', '--format=%T', tmp_path]) # noqa: E501
+        #       to not be nfs. This is only valid for Linux and we'd like to
+        #       stay at least also OSX compatible
+        tmp_path = env.get('SPACK_TMPDIR', '/tmp/spack') + '/tf'
+        mkdirp(tmp_path)
+        env['TEST_TMPDIR'] = tmp_path
+        env['HOME'] = tmp_path
+        env['CC'] = env['SPACK_CC']
+        env['CXX'] = env['SPACK_CXX']
+
+        configure()
+
+        # version dependent fixes
+        if self.spec.satisfies('@1.3.0:1.5.0'):
+            # checksum for protobuf that bazel downloads (@github) changed,
+            # comment out to avoid error better solution: replace wrong
+            # checksums in workspace.bzl
+            # wrong one: 6d43b9d223ce09e5d4ce8b0060cb8a7513577a35a64c7e3dad10f0703bf3ad93,                  # noqa: E501
+            # online: e5fdeee6b28cf6c38d61243adff06628baa434a22b5ebb7432d2a7fbabbdb13d                      # noqa: E501
+            filter_file(r'sha256 = "6d43b9d223ce09e5d4ce8b0060cb8a7513577a35a64c7e3dad10f0703bf3ad93"',     # noqa: E501
+                        r'#sha256 = "6d43b9d223ce09e5d4ce8b0060cb8a7513577a35a64c7e3dad10f0703bf3ad93"',    # noqa: E501
+                        'tensorflow/workspace.bzl')
+            # starting with tensorflow 1.3, tensorboard becomes a dependency
+            # (...but is not really needed? Tensorboard should depend on
+            # tensorflow, not the other way!)
+            # -> remove from list of required packages
+            filter_file(r"'tensorflow-tensorboard",
+                        r"#'tensorflow-tensorboard",
+                        'tensorflow/tools/pip_package/setup.py')
+        if self.spec.satisfies('@1.5.0:'):
+            # google cloud support seems to be installed on default, leading
+            # to boringssl error manually set the flag to false to avoid
+            # installing gcp support
+            # (https://github.com/tensorflow/tensorflow/issues/20677#issuecomment-404634519)  # noqa: E501
```

Comment:
Don't we have a variant for this?

Reply:
probably, but there are a ton of backend options, so we need someone who actually has the option to test this...